logger.go

```diff
@@ -122,6 +123,8 @@ func (l *Logger) ServeHTTP(rw http.ResponseWriter, r *http.Request, next http.Ha
 }
 
 func (l *Logger) responseLoggerLoop(ctx context.Context) {
+	defer recovery.LogStackTraceAndContinue("logger loop")
```

Comment:
I think this should be more like:
```
	defer func() {
		if err := recovery.HandlePanicWithError(recover(), nil, "logger loop"); err != nil {
			grip.Error(message.WrapError(err, message.Fields{
				"message": "panic in logger loop",
			}))
		}
	}()
```

Reply:
Is there an advantage to that particular function? It seemed to me that `LogStackTraceAndContinue` was more consistent with other cases of recovery around logkeeper. They will have mostly the same effect: they both log a message like
```go
grip.Alert(message.Fields{
	"message": "hit panic; adding error",
	"op": "logger loop",
	"panic": <panic_error>.Error(),
	"stack": <the stack...>,
})
```

It seems like the only difference is that `HandlePanicWithError` will also log a _second_ message.

Reply:
You are right, idk why we don't use this everywhere it is simpler!

Reply:
thanks! 😄